app/code/Magento/Wishlist/Block/Customer/Wishlist/Item/Column/Review.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+/**
+ * Review item column in customer wishlist table
+ */
+namespace Magento\Wishlist\Block\Customer\Wishlist\Item\Column;
+
+/**
+ * @api
+ */
+class Review extends \Magento\Wishlist\Block\Customer\Wishlist\Item\Column
```

Comment:
Please, remove the empty class and use Magento\Wishlist\Block\Customer\Wishlist\Item\Column as a block class directly in layout.

Reply:
@rogyar I have created empty class for similarity with all other columns

Reply:
@rogyar I think other empty classes should be marked as `@deprecated` too. Please confirm

Reply:
Thank you for the change. 
I meant removing only the `Review` class. There's no need to change the existing layout records since it may bring backward compatibility issues (for example, if the existing classes are used within a scope of some Magento setup). Also, the existing classes are not actually deprecated. Remove this annotation, please.
The main idea is to avoid bringing a new empty class and leave the existing as they are (at least, for now). 
Thanks.

Reply:
@rogyar Deprecation of class and removing usage from layout don't break any installations.
I think that we should use similar realization for all blocks.
In fact empty class allow to use plugins for customizations and I prefer to add empty class for review

Reply:
I agree regarding the deprecation. 
As for the backward compatibility, check the following scenario. Bad scenario but believe me, fair enough for more cases that we can expect. 
An existing store uses the original layout but overrides the blocks (using `preference` in `di.xml`). As result, the overridden blocks are not used anymore (they are not referenced in layout) and the store behavior is quite unpredictable.

Reply:
I agree, layout changes were hasty. Changes for layout is reverted